### ISSUE TITLE
Questions about G1M skinning！

### ISSUE THREAD

**Issue by qq1035732175** [Joschuka/Project-G1M#15]
I extracted the model animation texture of Dynasty Warriors 8 using Steven's Gas Machine tool, but there are some errors when using the G1M script,
Both the old py script and the latest dll have been tested, and the whole series of "dynasty warriors 8 empires" "DYNASTY WARRIORS 8: Xtreme Legends" of this game have this problem,

![VYIsgIX](https://user-images.githubusercontent.com/58507193/163724159-70f3615e-d76e-423d-ad6d-43493533bb72.jpg)
![Eqo2emV](https://user-images.githubusercontent.com/58507193/163724161-ffd5de73-40c0-453b-995b-959f7b4a8b21.jpg)

I will upload some model texture animation samples

[https://drive.google.com/file/d/1odQhcjuL1qpSZPRfVEqGToYe6JLfkdbb/view?usp=sharing](url)

**Comment by Joschuka** [comment]
Yeah looks like a bug indeed, I'll check it out when I have the time.
